docs/tutorial/crud.md

```diff
@@ -0,0 +1,139 @@
+# CRUD Operations
+
+CRUD is an acronym for Create, Read, Update, and Delete. These operations may be
+performed via the [MongoDB\Collection][collection] class, which implements
+MongoDB's cross-driver [CRUD specification][crud-spec]. This page will
+demonstrate how to insert, query, update, and delete documents using the
+library. A general introduction to CRUD operations in MongoDB may be found in
+the [MongoDB Manual][crud].
+
+[collection]: ../classes/collection.md
+[crud-spec]: https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst
+[crud]: https://docs.mongodb.org/manual/crud/
+
+## Querying
+
+### Finding One Document
```

Comment:
Does any mention want to be made of [Persistable](http://php.net/mongodb-bson-persistable) and how `_pclass` is used when an record is unserialized?